test/server.js

```diff
@@ -17,6 +17,12 @@ const env = require('./environment');
 
 const testOutputPath = `${__dirname}/test_reports/junit/e2e-results.xml`;
 
+const PORT = process.env.PORT || 3000; // 80/443 standard http/https
```

Comment:
will probably need update the README to make users aware that they can override the port number

Reply:
Agree, along with the `KEY_PATH` and `CERT_PATH`, if we want to keep those env vars.

Reply:
Also due to the `'ReferenceError: process is not defined'` issue we should move setting PORT  envVar into `environment.js`

Reply:
I'm confused why it matters if its here in `server.js` or in `environment.js`... thats strange.

But agree, I'll move it over, or just eliminate it?  Not sure if it should be configurable anyway...